### PR TITLE
[CFM-1133] -Update group title (reviewed)

### PR DESCRIPTION
**Pull request by ybaras**
https://issuetracker.amplexor.com/jira/browse/CFM-1133

### CODE REVIEW

**Comment by ybaras** [comment]
test-record

**Comment by ybaras** [comment]
**Smartphone:**

green
orange
red

**Comment by ybaras** [comment]
@ordavidil Ready for review.

File: project/profiles/capacity4more/modules/c4m/content/c4m_content_group/c4m_content_group.block.inc [hunk 1]
```diff
@@ -155,9 +155,9 @@ function _c4m_content_group_block_header_name_banner() {
   elseif (isset($wrapper->group_access)) {
     // Determine access per group_access parameter.
     $group_access = $wrapper->group_access->value() ? 'private' : 'public';
-    $variables['group_access'] = $group_access;
   }
 
+  $variables['group_access'] = $group_access;
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/content/c4m_content_group/c4m_content_group.block.inc]: @ybaras 
Let's just use `c4m_og_get_access_type()` in here and remove all the unnecessary logic from this function ...

File: project/profiles/capacity4more/modules/c4m/content/c4m_content_group/c4m_content_group.block.inc [hunk 2]
```diff
@@ -97,11 +97,9 @@ function _c4m_content_group_block_header_name_banner() {
   $variables['organisation_icons'] = array();
   $variables['organisation_ellipsis'] = FALSE;
   $variables['emails'] = array();
+  $variables['group_access'] = c4m_og_get_access_type($group)['type'];
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/content/c4m_content_group/c4m_content_group.block.inc]:
```php
$group_access = c4m_og_get_access_type($group);
$variables['group_access'] = $group_access['type'];
```

File: project/profiles/capacity4more/modules/c4m/content/c4m_content_group/c4m_content_group.block.inc [hunk 3]
```diff
@@ -97,11 +97,9 @@ function _c4m_content_group_block_header_name_banner() {
...
 
-  if (isset($wrapper->pluggable_node_access)
-    && !empty($wrapper->pluggable_node_access->value())) {
-    // If pluggable_node_access is set, then access is restricted.
-    $group_access = 'restricted';
+  if ($variables['group_access'] == 'restricted') {
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/content/c4m_content_group/c4m_content_group.block.inc]:
```php
if ($group_access['type'] == 'restricted') {
```

File: project/profiles/capacity4more/themes/c4m/kapablo/build/src/stylesheets/components/_groups.scss [hunk 4]
```diff
@@ -546,6 +545,12 @@
   float: left;
   margin: 0 0 0 18px;
 
+  &--access {
```
Comment (ordavidil) [inline project/profiles/capacity4more/themes/c4m/kapablo/build/src/stylesheets/components/_groups.scss]: Seems wrong

**Comment by jover** [comment]
See comments on [CFM-1133](https://issuetracker.amplexor.com/jira/browse/CFM-1133)

**Comment by ybaras** [comment]
**1024px:**



**768px:**



**480px:**

**Pull request merged**